onnxruntime/core/providers/webnn/builders/impl/unary_op_builder.cc

```diff
@@ -49,18 +63,30 @@ Status UnaryOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const
   return Status::OK();
 }
 
+// All unary ops except "Not" and "Identity" contain 'consumed_inputs' attribute before opset 6,
+// which is not supported in WebNN EP for now.
+int UnaryOpBuilder::GetMinSupportedOpSet(const Node& node) const {
+  if (node.OpType() != "Not" && node.OpType() != "Identity")
+    return 6;
+}
+
 void CreateUnaryOpBuilder(const std::string& op_type, OpBuilderRegistrations& op_registrations) {
   if (op_registrations.op_builder_map.find(op_type) != op_registrations.op_builder_map.cend())
     return;
 
   static std::vector<std::string> op_types =
       {
+          "Ceil",
           "Cos",
           "Erf",
+          "Exp",
           "Floor",
+          "Identity",
           "Not",
+          "Reciprocal",
           "Sin",
           "Sqrt",
+          "Tan"
```

Comment:
lint pass in the CI wants it to be
```
"tan");
```

Reply:
guschmue: 🤨 ORT's linter rules should at a minimum be self-consistent. e.g. See line 30 below in op_builder_factory.cc where the closing curly parenthesis is properly symmetrically placed on a new line. If ORT is complaining about this, then zesongw can unblock his CR by appeasing the broken rule, but the rule remains broken and should be fixed.